org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/uriresolver/CacheResourcesManager.java

```diff
@@ -167,13 +167,15 @@ public Path getResource(final String resourceURI) throws IOException {
 				String actualURI = resourceURI;
 				URL url = new URL(actualURI);
 				conn = url.openConnection();
+				conn.setRequestProperty("User-Agent", "LemMinX");

```

Comment:
Use constants like USER_AGENT_KEY = "User-Agent" and USER_AGENT_VALUE = "LemMinX"